[2024-12-16T21:09:39.618+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_name_dag.load_data manual__2024-12-16T21:04:35.517399+00:00 [queued]>
[2024-12-16T21:09:39.626+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_name_dag.load_data manual__2024-12-16T21:04:35.517399+00:00 [queued]>
[2024-12-16T21:09:39.626+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2024-12-16T21:09:39.638+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-16 21:04:35.517399+00:00
[2024-12-16T21:09:39.642+0000] {standard_task_runner.py:57} INFO - Started process 9288 to run task
[2024-12-16T21:09:39.644+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'full_name_dag', 'load_data', 'manual__2024-12-16T21:04:35.517399+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/full_name_dag.py', '--cfg-path', '/tmp/tmpisbtzzgd']
[2024-12-16T21:09:39.646+0000] {standard_task_runner.py:85} INFO - Job 107: Subtask load_data
[2024-12-16T21:09:39.662+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T21:09:39.697+0000] {task_command.py:415} INFO - Running <TaskInstance: full_name_dag.load_data manual__2024-12-16T21:04:35.517399+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T21:09:39.776+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_name_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T21:04:35.517399+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-16T21:04:35.517399+00:00'
[2024-12-16T21:09:39.813+0000] {base.py:73} INFO - Using connection ID 'admin' for task execution.
[2024-12-16T21:09:41.693+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T21:09:41.706+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=full_name_dag, task_id=load_data, execution_date=20241216T210435, start_date=20241216T210939, end_date=20241216T210941
[2024-12-16T21:09:41.742+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T21:09:41.779+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
